packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20240321</version>
+    <version>0.0.0.20240402</version>
```

Comment:
You need to rebase as the version is the same as in https://github.com/mandiant/VM-Packages/pull/962/files#diff-e403a7afac0855f61eaed356192afac07ee3faf31d423f5e9afd8318ee50c188R5 recently merged